src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__content">
+    <div class="product-card" data-qa="card">
+      <div class="product-card__content">
+        <div class="product-card__image-blok">
+          <img 
+            class="product-card__img" 
+            src="./images/imac.jpeg"
+            alt="img__product"
+          >
+        </div>
+        <div class="product-card__info">
+          <p class="product-card__name">
```

Comment:
```suggestion
          <h2 class="product-card__name">
```
це, швидше, заголовок